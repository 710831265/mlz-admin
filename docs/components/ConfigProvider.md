# ConfigProvider 全局化配置

**为组件提供统一的全局化配置。**
