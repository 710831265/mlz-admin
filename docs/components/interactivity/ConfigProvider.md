# ConfigProvider 全局化配置

**为组件提供统一的全局化配置。**

### 基本使用

<code src="./../../demo/ConfigProvider/normal-usage.demo.tsx" />
