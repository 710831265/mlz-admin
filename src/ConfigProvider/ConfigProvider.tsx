import React from 'react';
import { ConfigProvider as AntdConfigProvider } from 'antd';

const ConfigProvider = AntdConfigProvider;

export default ConfigProvider;
