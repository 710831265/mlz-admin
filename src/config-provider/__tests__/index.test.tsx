import React, { Component } from 'react';
import { testMount, testSnapshot } from '../../../tests';
import enUS from 'antd/es/locale/en_US';
import ConfigProvider from '..';

describe('ðŸ§ª ConfigProvider', () => {
  testMount(ConfigProvider, {
    locale: enUS,
  });
  testSnapshot(ConfigProvider, {
    staticRendered: true,
    locale: enUS,
  });
});
