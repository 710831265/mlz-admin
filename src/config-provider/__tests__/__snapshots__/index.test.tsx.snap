// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`🧪 ConfigProvider snapshot test 快照匹配 1`] = `""`;
